apps/andi/lib/andi_web/live/edit_live_view/extract_steps/extract_auth_step_form.ex

```diff
@@ -40,36 +40,36 @@ defmodule AndiWeb.ExtractSteps.ExtractAuthStepForm do
 
             <div class="extract-auth-step-form__destination">
               <%= label(f, :destination, DisplayNames.get(:destination),  class: "label label--required") %>
-              <%= text_input(f, :destination, class: "input") %>
+              <%= text_input(f, :destination, id: "step_#{@id}__auth_destination", class: "input") %>
```

Comment:
apparently having duplicate ids on liveview elements caused the issue where new steps of the same type would blank out the fields of old steps